dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/AppsResourceTest.java

```diff
@@ -363,6 +367,95 @@ public void Test_Create_App_descriptor_Then_Create_App_Integration_Then_Delete_O
         }
     }
 
+    /**
+     * Test we save a secret that is described as 1 public value and 2 hidden ones
+     * Given scenario: We create a secret then we update the non-hidden (public) values and send a string of asters like the UI would
+     * Expected Result: Only the public value  was supposed to be updated. The other 2 hidden fields should remain the same since we sent a bunch of *****
+     * @throws IOException
+     */
+    @Test
+    public void Test_Create_App_descriptor_Then_Create_App_Integration_Then_Update_Secrets()
```

Comment:
Shouldn't we have also a Postman test for this one?

Reply:
Well for one.. the validation I perform here can not be done from postman.

Reply:
Sure makes sense. What we need is some postman coverage against these resources. Doesn't have to be with this particular case. Because if someone changes the endpoint path in code, an integration won't catch that... because it hits the java class directly.